serverless.yml

```diff
@@ -19,10 +20,23 @@ provider:
             - "logs:*"
           Resource: "*"
 
+  ecr:
+    # In this section you can define images that will be built locally and uploaded to ECR
+    images:
+      poprox-recommender:
+        path: "./"
+
 plugins:
   - serverless-offline
   - serverless-python-requirements
 custom:
+  "offline:true": # offline enabled parameters
+    enabled: true
+    filePrefix: "offline."
+  "offline:false": # offline disabled parameters
+    enabled: false
+    filePrefix: ""
+  offline: ${self:custom.offline:${strToBool(${env:IS_OFFLINE, false})}}
```

Comment:
This sets the values of `${self:custom.offline}` to one or the other of the sets of values defined in the preceding 6 lines